jest.config.ts

```diff
@@ -1,4 +1,4 @@
-const { getJestProjects } = require('@nrwl/jest');
+const { getJestProjects } = require('@nx/jest');
 
 export default {
   projects: getJestProjects(),
```

Comment:
Nx version change should have change this to use `getJestProjectsAsync`

Reply:
yeah, it didn't :D